sites/public/pages/index.tsx

```diff
@@ -131,3 +206,16 @@ export default function Home() {
     </Layout>
   )
 }
+
+export async function getStaticProps() {
+  let latestListings = []
+  try {
+    const response = await axios.get(
+      `${process.env.listingServiceUrl}?limit=5&orderBy=mostRecentlyUpdated&availability=hasAvailability`
```

Comment:
Consider separating out the params for readability - see `hooks.ts` for example.